server/src/completions.ts

```diff
@@ -0,0 +1,563 @@
+  },
+  {
+    label: 'default()',
+    kind: CompletionItemKind.Property,
+    detail: '@default(_ expr: Expr)',
+    documentation: 'Specifies a default value if null is provided.',
+  },
+  {
+    label: 'relation()',
+    kind: CompletionItemKind.Property,
+    detail:
+      '@relation(_ name?: String, references?: Identifier[], onDelete?: CascadeEnum)\nArguments:\n•name: (optional, except when required for disambiguation) defines the name of the relationship. The name of the relation needs to be explicitly given to resolve amibiguities when the model contains two or more fields that refer to the same model (another model or itself).\n•references: (optional) list of field names to reference',
+    documentation:
+      'Specifies and disambiguates relationships when needed. Where possible on relational databases, the @relation annotation will translate to a foreign key constraint, but not an index.',
+  },
+]
+
+const supportedDataSourceFields: CompletionItem[] = [
+  {
+    label: 'provider',
+    kind: CompletionItemKind.Field,
+    documentation:
+      'Can be one of the following built in datasource providers:\n•`postgresql`\n•`mysql`\n•`sqlite`',
+  },
+  {
+    label: 'url',
+    kind: CompletionItemKind.Field,
+    documentation:
+      'Connection URL including authentication info. Each datasource provider documents the URL syntax. Most providers use the syntax provided by the database. (more information see https://github.com/prisma/specs/blob/master/schema/datasource_urls.md)',
+  },
+]
+
+const supportedGeneratorFields: CompletionItem[] = [
+  {
+    label: 'provider',
+    kind: CompletionItemKind.Field,
+    documentation:
+      'Can be a path or one of the following built in datasource providers:\n•`prisma-client-js`\n•`prisma-client-go` (This is not implemented yet.)',
+  },
+  {
+    label: 'output',
+    kind: CompletionItemKind.Field,
+    documentation: 'Path for the generated client.',
+  },
+  {
+    label: 'platforms',
+    kind: CompletionItemKind.Field,
+    detail: 'Declarative way to download the required binaries.',
+    documentation:
+      '(optional) An array of binaries that are required by the application, string for known platforms and path for custom binaries.',
+  },
+  {
+    label: 'pinnedPlatform',
+    kind: CompletionItemKind.Field,
+    detail: 'Declarative way to choose the runtime binary.',
+    documentation:
+      '(optional) A string that points to the name of an object in the platforms field, usually an environment variable.\nWhen a custom binary is provided the pinnedPlatform is required.',
+  },
+]
+
+function toCompletionItems(
+  allowedTypes: string[],
+  kind: CompletionItemKind,
+): CompletionItem[] {
+  const items: CompletionItem[] = []
+  allowedTypes.forEach((type) => items.push({ label: type, kind: kind }))
+  return items
+}
+
+function getSuggestionForBlockAttribute(blockType: string): CompletionItem[] {
+  if (blockType != 'model') {
```

Comment:
I would personally replace all `!=` with `!==` for consistency. https://github.com/airbnb/javascript#comparison--eqeqeq